docs/section9/Rule6-aa.md

```diff
@@ -0,0 +1,57 @@
+
+# Lighting - Rule 6-aa
+
+**Rule ID:** 6-aa  
+**Rule Description:** Proposed building is modeled with other programmable lighting controls through a 10% schedule reduction.  
+**Rule Assertion:** Proposed RMR = expected value  
+**Appendix G Section:** Lighting  
+**Appendix G Section Reference:**  
+
+- Table G3.7, Performance Rating Method Lighting Power Density Allowances and Occupancy Sensor Reductions Using the Space-by-Space Method  
+
+**Applicability:** All required data elements exist for P_RMR and B_RMR  
+**Applicability Checks:**  
+
+  1. Building total area is less than 5,000sq.ft.  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+
+## Rule Logic:  
+
+- For each building_segment in the Proposed model: ```For building_segment in P_RMR.building.building_segments:```  
+  - Get thermal_block from building segment: ```thermal_block in building_segment.thermal_blocks:```
+  - Get thermal_zone from thermal block: ```thermal_zone in thermal_block.zones:```
+  - Get space from thermal zone: ```space in thermal_zone.spaces:```
+  - Get floor_area from space: ```space_floor_area = space.floor_area```
+    - Get building total floor area: ```building_total_area_proposed = sum(space_floor_area)```
+
+- **Applicability Check 1:**```if building_total_area_proposed < 5000```
+
+  - Get purpose_type from interior lighting: ```purpose_type =  interior_lighting.purpose_type:```  
```

Comment:
purpose_type is just General, Task, Decorative. I think you want space.lighting_space_type

Reply:
this issue remains

Reply:
Updated in #39